src/scripts/modules/media/footer/footer.less

```diff
@@ -37,8 +37,16 @@
           content: "+";
         }
       }
+      
+      &:focus {
+        border-top: 0.1rem solid @brand-primary;
+        border-left: 0.1rem solid @brand-primary;
+        border-right: 0.1rem solid @brand-primary;
+        outline: 0;
+      }
 
-      &.active {
+      &.active,
+      &.active>button {
```

Comment:
Just a style comment, most of the rest of the CSS/LESS uses whitespace between selector symbols, ie: `&.active > button`.